scipy/special/_add_newdocs.py

```diff
@@ -4155,13 +4155,14 @@ def add_newdoc(name, doc):
     """
     gammainccinv(a, y)
 
-    Inverse of the upper incomplete gamma function with respect to `x`
+    Inverse of the regularized upper incomplete gamma function with
+    respect to `x`
```

Comment:
How about we delete "with respect to x" here (and also in the `gammaincinv` docstring below? There's a couple of reasons: it's confusing because there is no input `x`, it doesn't add anything useful to what's written further down, and the single line is now too long which makes the summary table content overflow.

Reply:
Hmm, it *does* seem that most descriptions of inverse methods in `scipy.special` do mention 'with respect to `<var>`", where `<var>` is a parameter of the method to be inversed. In case of `gammaincc(a, x)`, the original method has two parameters: the inverse may be with respect to either `a` or `x`. However, the "with respect to" part is definitely the least important part of the description here, and it is described later in the long description anyway.

so tldr; I agree, added in ed587fb097eeceb1cc8c804a659cc18347b28f22 :)